src/puppetlabs/puppetdb/queue.clj

```diff
@@ -60,13 +125,6 @@
 
 (def parse-metadata (metadata-parser))
 
-(def parse-cmd-filename
-  (let [parse-metadata (metadata-parser (cons "unknown" metadata-command-names))]
-    (fn [s]
-      (let [rx #"([0-9]+)-(.*)"]
-        (when-let [[_ id qmeta] (re-matches rx s)]
-          (parse-metadata qmeta))))))
```

Comment:
@rbrw FWIW, keeping "unknown" a DLO thing is part of why I moved this function over to the DLO namespace.